src/Model/common.hpp

```diff
@@ -158,22 +158,26 @@ void seissol::model::getTransposedElasticGodunovState( Material const&
     QgodLocal(idx,idx) += 1.0;
   }
   
-  applyBoundaryConditionToElasticFluxSolver(faceType, QgodNeighbor);
+  applyBoundaryConditionToElasticFluxSolver(faceType, QgodLocal);
```

Comment:
Should we set QGodNeighbor to zero or NaN? In the first case, the solver would still work if QGodNeighbor is used by accident, in the second case, it would immediately fail such that the cause can be (quickly?) found?

Reply:
I'd vote for signaling NaNs here. You can activate floating point exceptions and then SeisSol would crash immediately if it would be used by accident.